<?xml version="1.0"?>
<launch>
	<arg name="x_max" default="9"/>
	<arg name="y_max" default="9"/>
	<arg name="z_max" default="3"/>
	<arg name="x_min" default="-1"/>
	<arg name="y_min" default="-1"/>
	<arg name="z_min" default="0"/>
	<arg name="file" default="/home/pedrobelo/Desktop/data123.txt"/>
	<arg name="octomapTopic" default="/octomap_full"/>

	<node name="eval" pkg="evaluation" type="evaluation" output="screen">
		<param name="x_max" value="$(arg x_max)"/>
		<param name="y_max" value="$(arg y_max)"/>
		<param name="z_max" value="$(arg z_max)"/>
		<param name="x_min" value="$(arg x_min)"/>
		<param name="y_min" value="$(arg y_min)"/>
		<param name="z_min" value="$(arg z_min)"/>
		<param name="file" value="$(arg file)"/>
		<param name="octomapTopic" value="$(arg octomapTopic)"/>
	</node>
</launch>
